torchreid/engine/engine.py

```diff
@@ -205,51 +239,74 @@ def run(
             self.writer = SummaryWriter(log_dir=log_dir)
 
         # Save zeroth checkpoint
-        self.save_model(-1, save_dir)
+        if self.save_chkpt and not lr_finder:
```

Comment:
I think we can get rid of saving this checkpoint and the corresponding parameter.

Reply:
Do you propose deleting the saving zeroth checkpoint?